yt/visualization/fits_image.py

```diff
@@ -251,7 +260,7 @@ def __init__(self, data, fields=None, length_unit=None, width=None,
                         hdu.header[key] = float(value.value)
                         hdu.header.comments[key] = "[%s]" % value.units
                 if self.current_time is not None:
-                    hdu.header["time"] = self.current_time
+                    hdu.header["time"] = float(self.current_time.value)
```

Comment:
I might just be confused here, so sorry in advance (and it's a little bit outside of the relevance of this PR). 

In the previous few lines you set the `time` header with `value.value`, and then if `current_time` exists you overwrite `time` with that value. Is that right? Would there be a case where `time` exists but `current_time` does not? Can we always assume that the units of `current time` are the same as `time`?

Reply:
Sorry, this code isn't as well-documented as it should be. That should be my next job. :)

The value that is set above by `value.value` is `header["TUNIT"]`, whereas `header["TIME"]` is set by the `self.current_time` value.

Actually, I don't think that `self.current_time` will ever be `None` given the way I've set things up. So I should probably not have this conditional here.